srcpkgs/dragon-player/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'dragon-player'
+pkgname=dragon-player
+version=18.08.0
+revision=1
+build_style=cmake
+wrksrc="${pkgname%-*}-${version}"
+hostmakedepends="extra-cmake-modules pkg-config qt5-qmake qt5-host-tools kdoctools"
+makedepends="kio-devel kparts-devel"
+depends="virtual?phonon-qt5-backend hicolor-icon-theme"
+short_desc="is a multimedia player where the focus is on simplicity, instead of features."
```

Comment:
`"Multimedia player with focus on simplicity, instead of features"` Shorter and xlint shouldn't complain about it.

Reply:
looks good to me, even if I'm not a great fan of KDE. :roll_eyes: